src/fix_gemc.cpp

```diff
@@ -0,0 +1,514 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+// Gen Questions:
+// 1) What is a tagint?
+// 2) What are all the nullptrs in the definition of a fix
```

Comment:
For all new and refactored code we initialize all class members that are pointers to `nullptr` so you can safely use `memory->destroy()` or `delete[]` on them, even if they have not yet been allocated. When a class is allocated and the memory allocator "recycles" previously used address space, it is not easily possible to tell if the memory has been allocated or not (malloc on Linux is special, that first allocations are all zero bytes so people tend to depend on this, but that is dangerous and can lead to all kinds of problems like use after free or memory corruption or segmentation faults).  For code paths that may be passed multiple times, this simplifies the code by just deleting/destroying it always and thus memory leaks are avoided.